tests/test_transforms/test_decomposition/test_binseg_trend_transform.py

```diff
@@ -17,9 +17,19 @@
 from etna.transforms.decomposition import BinsegTrendTransform
 
 
+@pytest.fixture
+def ts_with_nans(df_with_nans) -> TSDataset:
```

Comment:
I guess we actually need to change the entire fixture and mark all the places where it is used with xfail, except the tests connected to this PR